### DISCUSSION TITLE
Change theme color by code

### DISCUSSION THREAD

**Discussion by espdav** [Goodwy/PlayBooks#45] [Ideas]
The hexadecimal code is unclickable, i wonder if it's possible to make it clickable to insert custom color code.

![Screenshot_20230713-064339.png](https://github.com/Goodwy/PlayBook/assets/85557567/4fbe85e2-5a2a-45ec-a874-f8070fb55fe2)

**Comment by Goodwy** [comment]
There is no need to set the color you want. The color you specify is not used directly. This is the new android color system, where a color is passed in and a set of colors of the whole system is created based on it. According to my observations, this system does not give a great variety of colors in the output.
